src/core/crypto/secp256k1.ts

```diff
@@ -149,11 +149,12 @@ export class Secp256k1PrivateKey extends Serializable implements PrivateKey {
    * Create a new PrivateKey instance from a Uint8Array or String.
    *
    * @param hexInput A HexInput (string or Uint8Array)
+   * @param strict If true, private key must AIP-80 compliant.
    */
-  constructor(hexInput: HexInput) {
+  constructor(hexInput: HexInput, strict?: boolean) {
     super();
 
-    const privateKeyHex = Hex.fromHexInput(hexInput);
+    const privateKeyHex = PrivateKey.parseHexInput(hexInput, "secp256k1", strict);
```

Comment:
lets have global consts for `secp256k1` and `ed25519`. can add it in the [types file](https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/types/index.ts#L105)